scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -18,6 +18,17 @@ import {addInternalEventListener} from 'core/internal-events';
 import {validateMediaFieldsThrows} from '../controllers/ChangeImageController';
 import {getLabelNameResolver} from 'apps/workspace/helpers/getLabelForFieldId';
 
+export function getErrorMessage(message, notify) {
+    return getLabelNameResolver().then((getLabelForFieldId) => {
+        const field = message.split(' ')[0];
+        const fieldLabel = getLabelForFieldId(field.toLocaleLowerCase());
+        const text = message.substr(message.indexOf(' ') + 1);
+
+        notify.error(gettext(`{{field}} ${text}`, {field: fieldLabel.toLocaleUpperCase()}));
```

Comment:
`gettext` should only be used on static strings. I see that you have added string extraction separately, but it's a hack since strings that are coming from the back-end should already come translated. Could this be done?